components/challenge_bypass_ristretto/batch_dleq_proof.cc

```diff
@@ -224,7 +224,7 @@ base::expected<BatchDLEQProof, std::string> BatchDLEQProof::DecodeBase64(
       cbr_cxx::decode_base64_batch_dleq_proof(encoded));
 
   if (!raw_proof_result->is_ok()) {
-    return base::unexpected(raw_proof_result->error().msg.data());
```

Comment:
Missed this somehow during a review/implementation. All error messages in `challenge-bypass-ristretto` are hardcoded in C++ for now. 
Proper error messaging should be done here: https://github.com/brave/brave-browser/issues/35455